Lib/inspect.py

```diff
@@ -378,11 +378,16 @@ def isfunction(object):
     Function objects provide these attributes:
         __doc__         documentation string
         __name__        name with which this function was defined
+        __qualname__    fully qualified name of this function
```

Comment:
It's not "full", it doesn't include the module:

```suggestion
        __qualname__    qualified name of this function
```